### PR TITLE
Features: Aderyn Ignore Custom detectors

### PR DESCRIPTION
**Pull request by TilakMaddy**

### CODE REVIEW

File: tests/contract-playground/src/AderynIgnoreCustomDetectors.sol [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
```
Comment (alexroan) [inline tests/contract-playground/src/AderynIgnoreCustomDetectors.sol:2]:
```suggestion
pragma solidity 0.8.19;
```

**Review by alexroan — APPROVED** [review]
Bro this is excellent!

One small suggestion change and we're good to go.

File: tests/contract-playground/src/AderynIgnoreCustomDetectors.sol (hunk 1 above)
Comment (TilakMaddy) [reply tests/contract-playground/src/AderynIgnoreCustomDetectors.sol:2]: Any reason for this ? I'm missing context ..

Comment (TilakMaddy) [reply tests/contract-playground/src/AderynIgnoreCustomDetectors.sol]: ✅

**Pull request merged**